controllers/openshift_controller_test.go

```diff
@@ -631,6 +631,42 @@ var _ = Describe("OpenShift KataConfig Controller", func() {
 			By("Creating the KataConfig CR successfully")
 			Expect(k8sClient.Create(context.Background(), kataConfig)).Should(Succeed())
 
+			// Delete
+			By("Deleting KataConfig CR successfully")
+			kataConfigKey := types.NamespacedName{Name: kataconfig.Name}
+			Eventually(func() error {
+				k8sClient.Get(context.Background(), kataConfigKey, kataconfig)
+				return k8sClient.Delete(context.Background(), kataconfig)
+			}, timeout, interval).Should(Succeed())
+
+			By("Expecting to delete KataConfig CR successfully")
+			Eventually(func() error {
+				return k8sClient.Get(context.Background(), kataConfigKey, kataconfig)
+			}, timeout, interval).ShouldNot(Succeed())
+
+		})
```

Comment:
It is certainly a good thing to clean things up but I'd make this a separate fix for the `Custom KataConfig with CheckNodeEligibility create` test.

Reply:
Hi @gkurz, I'm not sure if I got your suggestion.

This section, that is deleting KataConfig after the test, is already in the test you mentioned. It is also duplicated on 3 other tests, I just copy and pasted as it was being done in the current file. Do you mind expand a little bit, please?


Note: I was planning to remove all those duplicated codes, maybe using ` AfterEach(func()`, but for some reason was not working. So I would like to address this in a different PR latter.